### PR TITLE
chromium: Forward port musl patches to 95.0.4638.69

### PR DESCRIPTION
**Pull request by kraj**
Signed-off-by: Khem Raj <raj.khem@gmail.com>

### CODE REVIEW

File: meta-chromium/recipes-browser/chromium/files/musl/fix-bits-stdint-intn-include.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- a/ui/ozone/platform/wayland/host/wayland_window.cc.orig
++++ b/ui/ozone/platform/wayland/host/wayland_window.cc
+@@ -4,7 +4,7 @@
+ 
+ #include "ui/ozone/platform/wayland/host/wayland_window.h"
+ 
+-#include <bits/stdint-intn.h>
++#include <stdint.h>
```
Comment (rakuco) [inline meta-chromium/recipes-browser/chromium/files/musl/fix-bits-stdint-intn-include.patch:8]: @msisov this looks like something that could be upstreamed. It's unusual to include headers in `bits/` rather than their counterparts in `/usr/include` directly. Was there a specific reason for this to happen here?

File: meta-chromium/recipes-browser/chromium/files/musl/chromium-95-harfbuzz-3.patch [hunk 2]
```diff
@@ -0,0 +1,31 @@
+From 9e00e702633c47c590a869bc66b5c2ceec09da50 Mon Sep 17 00:00:00 2001
+From: Dominik Röttsches <drott@chromium.org>
+Date: Tue, 28 Sep 2021 15:31:50 +0000
+Subject: [PATCH] Roll src/third_party/harfbuzz-ng/src/ 6602cbb70..a52c6df38 (80 commits)
```
Comment (rakuco) [inline meta-chromium/recipes-browser/chromium/files/musl/chromium-95-harfbuzz-3.patch:4]: This looks unrelated to musl. Can you explain what was breaking before?

File: meta-chromium/recipes-browser/chromium/files/musl/fix-bits-stdint-intn-include.patch (hunk 1 above)
Comment (kraj) [reply meta-chromium/recipes-browser/chromium/files/musl/fix-bits-stdint-intn-include.patch:8]: perhaps yes.

File: meta-chromium/recipes-browser/chromium/files/musl/chromium-95-harfbuzz-3.patch (hunk 2 above)
Comment (kraj) [reply meta-chromium/recipes-browser/chromium/files/musl/chromium-95-harfbuzz-3.patch:4]: I did not build with glibc so maybe it happens there too but with OE master which harfbuzz 3.1 I get runtime errors e.g.

```
browser log: |
        {"type":"error","text":"Error: Browser exited unexpectedly"}
        {"type":"error","text":"Non-zero exit code: 127"}
        {"type":"error","text":"Error relocating /usr/lib/chromium/chrome: hb_subset_input_set_retain_gids: symbol not found\nError relocating /usr/lib/chromium/chrome: hb_subset: symbol not found\nError relocating /usr/lib/chromium/chrome: hb_subset_input_set_drop_hints: symbol not found\n"}
```

Comment (rakuco) [reply meta-chromium/recipes-browser/chromium/files/musl/chromium-95-harfbuzz-3.patch:4]: I've just finished a chromium-x11 here on honister that did not throw any errors. The version of harfbuzz shipped in OE shouldn't matter, as we build with Chromium's own version (that's often very close to the tip of the tree). Is that not the case for you by any chance?

Also, is the change below enough to stop all those different relocation errors?

Comment (kraj) [reply meta-chromium/recipes-browser/chromium/files/musl/chromium-95-harfbuzz-3.patch:4]: yeah I had external harfbuzz in use, I am testing interrnal one right now if it works then this patch is not needed.

**Review by rakuco — APPROVED** [review]
lgtm

File: meta-chromium/recipes-browser/chromium/files/musl/fix-bits-stdint-intn-include.patch (hunk 1 above)
Comment (kraj) [reply meta-chromium/recipes-browser/chromium/files/musl/fix-bits-stdint-intn-include.patch:8]: submitted https://chromium-review.googlesource.com/c/chromium/src/+/3291865
